invision-app/app/components/moviesFound/moviesFound.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import './moviesFound.scss'
+import { connect } from 'react-redux';
```

Comment:
You do not need Redux and mapStateToProps function in this component. Please, clean files from the unused code